### PR TITLE
Added new graph Commit vs Runtime + Tests Run

### PR DESCRIPTION
**Pull request by xavierdelacruz**
- changed the csv URLS to use the actual data output directory

### CODE REVIEW

File: visualization-mock-data-output/runtime.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Commit,Runtime (s),Number of Tests Run
+1,42,95.1515s,,
+2,41,66.3931s
+3,1,9.5414s
+4,25,82.151s,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (lydiachoi) [inline visualization-mock-data-output/runtime.csv:5]: why are there so many commas in this one section?

Comment (xavierdelacruz) [reply visualization-mock-data-output/runtime.csv:5]: Oh these are test files, we wanted to test it out haha

Comment (xavierdelacruz) [reply visualization-mock-data-output/runtime.csv:5]: And the findings are, it doesn't matter how many commas, its the headers that decide the actual data that gets read into D3

**Review by lydiachoi — APPROVED** [review]

**Review by cboltt — APPROVED** [review]

**Comment by lydiachoi** [comment]
Also not a huge deal but can you squash the commits? The second commit is just renaming so if u squashed it it wouldn't clutter the commit history.

**Pull request merged**